helm/superset/templates/service-account.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ if .Values.serviceAccountName }}{{ .Values.serviceAccountName }}{{ else }}{{ include "superset.fullname" . }}{{ end }}
```

Comment:
Wouldn't that be enough to use:
{{ include "superset.serviceAccountName" . }}

Reply:
That's good idea, I've written whole conditional before working on `_helpers` :)
It's done!